app/models.py

```diff
@@ -1265,6 +1266,7 @@ def populate(cls):
                     type=row['type'],
                     title=row['title'],
                     content=row['content'],
+                    has_appeals_language=row['has_appeals_language'],
```

Comment:
Needs to be changed to `eval(row['has_appeals_language'])` or else populate function won't work.